resources/js/app.js

```diff
@@ -4,11 +4,11 @@
  * building robust, powerful web applications using Vue and Laravel.
  */
 
-window.autosize = require('autosize');
+window.autosize = require("autosize");
```

Comment:
We have a company-specific Prettier config:

https://github.com/tighten/handbook/tree/main/coding-standards/javascript